### ISSUE TITLE
Deploy Checklist: New Expensify 2022-04-25

### ISSUE THREAD

**Issue by francoisl** [Expensify/App#8777]
**Release Version:** `1.1.57-17`
**Compare Changes:** https://github.com/Expensify/App/compare/production...staging

**This release contains changes from the following pull requests:**

- https://github.com/Expensify/App/pull/7849
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8483
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8493
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8521
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8535
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8538
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8581
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8591
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8609
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8613
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8632
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8634
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8635
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8656
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8680
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8682
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8689
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8691
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8692
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8698
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8712
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8713
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8725
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8726
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8730
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8732
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8733
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8735
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8737
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8738
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8739
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8741
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8742
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8743
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8746
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8750
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8751
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8753
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8756
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8757
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8758
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8759
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8764
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8766
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8767
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8769
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8770
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8778
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8788
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8789
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8795
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8796
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8801
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8802
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8803
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8805
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8808
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8813
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8815
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8816
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8818
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8821
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8826
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8827
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8828
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8830
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8834
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8836
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8837
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8839
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8844
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8853
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8854
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8856
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8857
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8858
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8859
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8861
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8862
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8865
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8866
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8868
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8879
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8880
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8906
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8909
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8912
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8916
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8940
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8953
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8957
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8969
  - [x] QA
  - [x] Accessibility

- https://github.com/Expensify/App/pull/8970
  - [x] QA
  - [x] Accessibility

**Internal QA:**
- [x] https://github.com/Expensify/App/pull/8771 - @tylerkaraszewski @johnmlee101

**Deploy Blockers:**
- [x] https://github.com/Expensify/App/issues/8329
- [x] https://github.com/Expensify/App/issues/8783
- [x] https://github.com/Expensify/App/issues/8791
- [x] https://github.com/Expensify/App/issues/8800
- [x] https://github.com/Expensify/App/issues/8922
- [x] https://github.com/Expensify/App/issues/8924
- [x] https://github.com/Expensify/App/issues/8926
- [x] https://github.com/Expensify/App/issues/8945
- [x] https://github.com/Expensify/App/issues/8947
- [x] https://github.com/Expensify/App/issues/8962
- [x] https://github.com/Expensify/App/issues/8963
- [x] https://github.com/Expensify/App/issues/8968

**Deployer verifications:**
- [x] I checked the [App Timing Dashboard](https://graphs.expensify.com/grafana/d/yj2EobAGz/app-timing?orgId=1) and verified this release does not cause a noticeable performance regression.
- [x] I checked [Firebase Crashlytics](https://console.firebase.google.com/u/0/project/expensify-chat/crashlytics/app/android:com.expensify.chat/issues?state=open&time=last-seven-days&tag=all) and verified that this release does not introduce any new crashes.

cc @Expensify/applauseleads

**Comment by Julesssss** [comment]
I marked #8783 as NOT a blocker, as it's not that important overall. It'll get triaged as usual though.

**Comment by francoisl** [comment]
Hi @mvtglobally, how's it going with the checklist? Anything I can help with to move it forward?

**Comment by mvtglobally** [comment]
80%done. Will be updating the list in a little bit

**Comment by mvtglobally** [comment]
@francoisl  Team is still working through some of the PRs to complete all platforms coverage. 
So far logged Deploy Blocker - https://github.com/Expensify/App/issues/8791

**Comment by mvtglobally** [comment]
@francoisl Regression is completed 
https://github.com/Expensify/App/pull/8738#issuecomment-1109306672
https://github.com/Expensify/App/pull/8743 is failing due to #8787. Since its repro in PROD checking off 
We are re-testing https://github.com/Expensify/App/pull/8656 as per last message

**Comment by mvtglobally** [comment]
https://github.com/Expensify/App/pull/8656 is pass after retest. Checking off

**Comment by francoisl** [comment]
Cool, thanks. Going to follow up on #8738.
Additionally, #8682 and #8713 are also still unchecked, can you take a look please?

**Comment by mvtglobally** [comment]
#8682. We are facing issue with the last step. Rest seem ok. https://github.com/Expensify/App/pull/8682#issuecomment-1112821066 
https://github.com/Expensify/App/pull/8713 is failing https://github.com/Expensify/App/issues/8822. But since this is a new feature, I think we can check it off?

**Comment by marcaaron** [comment]
Found an issue with #8329 helping to resolve it in this PR https://github.com/Expensify/App/pull/8827

**Comment by Stutikuls** [comment]
Checked all PRs and created accessibility issues for all PRs where accessibility issue were found.
The non-applicable PRs are - #8538, #8613, #8635, #8656, #8680, #8691, #8713, #8726, #8732, #8738, #8746, #8767, #8803
logged Issues - #8521, #8535, #8632, #8634, #8682, #8692, #8698, #8712, #8730, #8743, #8751

**Comment by mvtglobally** [comment]
@sketchydroide 
Hope we can deploy soon. 2 Deploy Blockers still open
Based on this https://github.com/Expensify/App/pull/8682#issuecomment-1114952999 Checking off
https://github.com/Expensify/App/pull/8713 is failing https://github.com/Expensify/App/issues/8822. But since this is a new feature, I think we can check it off?

**Comment by sketchydroide** [comment]
> https://github.com/Expensify/App/pull/8713 is failing https://github.com/Expensify/App/issues/8822. But since this is a new feature, I think we can check it off?
@marcaaron and maybe @trjExpensify any input if we can check #8713?

**Comment by sketchydroide** [comment]
I will take a look at #8791 to see if we can unblock it

**Comment by trjExpensify** [comment]
>@marcaaron and maybe @trjExpensify any input if we can check https://github.com/Expensify/App/pull/8713?

Yeah, Marc closed #8822 last night as it's a feature in dev as part of the offline project.

**Comment by mvtglobally** [comment]
@sketchydroide pls tag me once https://github.com/Expensify/App/issues/8791 is ready to validate. 
https://github.com/Expensify/App/issues/8329 is not repro. not sure if internal fix was already pushed. 
Just waiting for these last 2 issues to get a new build

**Comment by marcaaron** [comment]
PR related to #8329 has been merged now, but doesn't seem like it was CP'd to staging. 

I'm not really understanding how it could not be reproducible when last tested if the code linked to the original PR made it to staging.

**Comment by marcaaron** [comment]
Not sure if 👆  is accurate as I got tagged on 👉  https://github.com/Expensify/App/pull/8871 and fixed some merge conflicts

**Comment by sketchydroide** [comment]
#8791 is still on slow progress, we might want to not make it a blocker since it seems rather hard to get to the end of it, and there is still the a way to go around it, the user can install the app and use it.

**Comment by mvtglobally** [comment]
@sketchydroide if you can push it, would be great. At this point we haven't had deploy in 10 days. 
Last build 1.1.57-5 hasn't deployed either.

**Comment by sketchydroide** [comment]
https://github.com/Expensify/App/pull/8871 should be fixes on the build above. @mvtglobally I think we can test it

@chiragsalian I will continue to investigate https://github.com/Expensify/App/issues/8791, I did some small progress last week, but it's still really a mistery what is happening, feel free to give it a try as well.

**Comment by mvtglobally** [comment]
@sketchydroide we are testing #8871. 
@chiragsalian Can you push the build or add new PRs today to this checklist. It has been almost 2 weeks since last deploy.

**Comment by chiragsalian** [comment]
> Can you push the build or add new PRs today to this checklist.

We unfortunately cannot unless we manually CP them to checklist. But i think this is valuable because we can discuss with the team if we should have a feature to deploy only to staging when needed. Seems like a double edge sword though since it would be harder to solve deploy blockers if they come up. Anyway shall discuss with the team. I'll check the checklist and issues today.

**Comment by chiragsalian** [comment]
@sketchydroide, was this CP by you to add around 30 tasks to the QA checklist intentional - https://github.com/Expensify/App/pull/8871?

Also how is it that there are two CP's referring the same PR (8827) but with very different file changes? I'm wondering if there was some bug here.
1. https://github.com/Expensify/App/pull/8871
2. https://github.com/Expensify/App/pull/8904

I feel like the first CP (8871) was incorrect which accidentally caused 30 issues being added to the checklist here. Let me know if i'm mistaken. 
Either way i've asked QA to go ahead and QA these items since it gives them something to do versus being stuck 🙂

**Comment by chiragsalian** [comment]
Checked off the following since it was just me testing stuff on staging.
https://github.com/Expensify/App/pull/8906
https://github.com/Expensify/App/pull/8909
https://github.com/Expensify/App/pull/8912
https://github.com/Expensify/App/pull/8916

**Comment by sketchydroide** [comment]
> @sketchydroide, was this CP by you to add around 30 tasks to the QA checklist intentional - https://github.com/Expensify/App/pull/8871?

No it was not intencional, to create those many new tasks, I just did the merge

I think one of the cherry picks was just to try to get the process working again, as it had been stuck with an error updating the version

**Comment by NikkiWines** [comment]
Checking off https://github.com/Expensify/App/issues/8922 as it appears to be a known issue with a [fix](https://github.com/Expensify/App/pull/7985) in the works.

**Comment by roryabraham** [comment]
It seems like something unexpected happened with this checklist. Why were so many new PRs added? Were they all CP'd or did we somehow deploy main to staging again?

**Comment by chiragsalian** [comment]
Oh btw discussed with rory 1:1. Many new PRs were added because of the CP of https://github.com/Expensify/App/pull/8871. I discussed it with andre too and it was not intentional and it looks like pushed up a lot of changes in that CP. Definitely not an intended affect but to make the best of the situation we'll continue to QA the new PR and do regression testing as well to ensure things are in a good place.

**Comment by chiragsalian** [comment]
Update. These blockers have been addressed and i've asked for a retest before checking them off,
https://github.com/Expensify/App/issues/8924
https://github.com/Expensify/App/issues/8926
https://github.com/Expensify/App/issues/8945

There's just one more blocker remaining and we'll address it tomorrow.

**Comment by mvtglobally** [comment]
@chiragsalian Regressions is almost complete. We will retest above fixes and update in the morning. 

Deploy Blockers 
https://github.com/Expensify/App/issues/8922
 https://github.com/Expensify/App/issues/8924
 https://github.com/Expensify/App/issues/8926
 https://github.com/Expensify/App/issues/8945
 https://github.com/Expensify/App/issues/8947

#8733 is failing due to https://github.com/Expensify/App/issues/8930 
#8758 is failing due to https://github.com/Expensify/App/issues/8932. not a blocker, checking off
#8778 is failing due to https://github.com/Expensify/App/issues/8956. not a blocker, checking off
#8826 is failing due to https://github.com/Expensify/App/issues/8951. not a blocker, checking off

**Comment by mvtglobally** [comment]
Checking off 
https://github.com/Expensify/App/issues/8926
https://github.com/Expensify/App/issues/8945

**Comment by thesahindia** [comment]
Just a heads up that we can check off https://github.com/Expensify/App/issues/8968
Because it won't be reproducible after https://github.com/Expensify/App/issues/8962 gets fixed.

**Comment by luacmartins** [comment]
Here's the [fix](https://github.com/Expensify/App/pull/8969) for https://github.com/Expensify/App/issues/8947

**Comment by chiragsalian** [comment]
The checklist isn't working too well for me

**Comment by chiragsalian** [comment]
The checklist is being a little buggy for me. Can you check or consider these done,
1. https://github.com/Expensify/App/pull/8957. (It has been tested from its deploy blocker issue)
2. https://github.com/Expensify/App/pull/8969. (It has been tested from its deploy blocker issue)
3. https://github.com/Expensify/App/issues/8968. (its a dupe of an earlier item)

I've asked for a retest of the following deploy blockers since the CP for them is now on staging. Once tested it can be checked off the list and the issue can be closed.
1. https://github.com/Expensify/App/issues/8947
2. https://github.com/Expensify/App/issues/8962

So the only thing remaining is https://github.com/Expensify/App/issues/8963 and we're currently working on it.

**Comment by mvtglobally** [comment]
@chiragsalian we are re-testing

**Comment by chiragsalian** [comment]
**Some more updates:**
1. https://github.com/Expensify/App/issues/8963 - No longer a blocker, can you check it off.
2. https://github.com/Expensify/App/pull/8771 - tested internally, so can you check it off.
3. https://github.com/Expensify/App/pull/8733 - Looks like its dependent on https://github.com/Expensify/App/issues/8930, since neither are really blockers we can check that off too.

**Comment by mvtglobally** [comment]
Checking off https://github.com/Expensify/App/issues/8962 https://github.com/Expensify/App/issues/8947 - retest Passed

**Comment by chiragsalian** [comment]
Awesome, thank you. Okay looks like we're all clear. Due to some important meetings today we're not deploying today but we'll deploy early tomorrow 🙂

**Comment by Stutikuls** [comment]
Checked all PRs on Production / Staging and created accessibility issues for all PRs where accessibility issues were found.
The non-applicable PRs are - #8742, #8753, #8756, #8759, #8766
Logged Issues - #7849, #8483, #8493, #8581, #8591, #8609, #8689, #8725, #8733, #8737, #8739, #8741, #8757, #8758, #8764, #8769, #8795, 
Note: [#8770] (Client: Currently there's no way for Applause to test our P2P payment flows, so you can ignore this for now as per the Client's message)

**Comment by chiragsalian** [comment]
:shipit:

**Comment by SumitDiyora** [comment]
Checked all PRs on Production / Staging and created accessibility issues for all PRs where accessibility issues were found.
The non-applicable PRs are - #8789, #8796, #8801, #8803, #8805, #8813, #8830, #8836, #8853, #8854, #8857, #8858, #8862, #8906, #8909, #8912, #8916
Logged Issues - #8778, #8788, #8795, #8802, #8808, #8813, #8815, #8816, #8818, #8826, #8827, #8828, #8834, #8837, #8839, #8844, #8856, #8859, #8866, #8868, #8879, #8940, #8953, #8957, #8969, #8970
